transport/src/main/java/io/netty/channel/nio/NioEventLoopGroup.java

```diff
@@ -84,6 +85,16 @@ public void rebuildSelectors() {
     @Override
     protected EventExecutor newChild(
             ThreadFactory threadFactory, Object... args) throws Exception {
-        return new NioEventLoop(this, threadFactory, (SelectorProvider) args[0]);
+        return new NioEventLoop(this, threadFactory, (SelectorProvider) args[0], backoffStrategy());
+    }
+
+    /**
+     * Backoff Strategy which is called every time a {@link #newChild(ThreadFactory, Object...)} is
+     * created. Can be overridden if a custom backoff strategy should be used.
+     *
+     * @return {@link NoopBackoffStrategy} as the default.
+     */
+    public BackoffStrategy backoffStrategy() {
+        return new NoopBackoffStrategy();
```

Comment:
@Scottmitch I considered that too, the only problem I had with this is that we need to call it N times per loop so one strategy isn't enough - we'd need some kind of factory and then I thought well this might be simpler.

I had the singleton for the Noop there already, I can add it back for sure :)